app/experimenter/experiments/tests/test_bugzilla.py

```diff
@@ -159,18 +182,38 @@ def test_api_error_logs_message(self):
         mock_response.status_code = 400
         self.mock_bugzilla_requests_post.return_value = mock_response
 
-        response_data = make_bugzilla_call("/url/", {})
+        response_data = make_bugzilla_post_call("/url/", {})
         self.assertEqual(response_data, mock_response_data)
 
     def test_request_error_passes_silently(self):
         self.mock_bugzilla_requests_post.side_effect = RequestException()
 
         with self.assertRaises(BugzillaError):
-            make_bugzilla_call("/url/", {})
+            make_bugzilla_post_call("/url/", {})
 
     def test_json_parse_error_passes_silently(self):
```

Comment:
This test isn't named properly (my fault).  I think it should say something like

    test_json_parse_error_raises_bugzilla_error